### DISCUSSION TITLE
[Question][Hw7] Why do we need CPath?

### DISCUSSION THREAD

**Discussion by kaist-kihwan** [prosyslab-classroom/cs348-information-security#729] [Q&A]
Name: Kim Kihwan

Since CPath does not occur in other rules, why do need them?
I got full score without using `derive_cpath1` and `derive_cpath2`.

**Comment by sujin0529** [comment]
Hi,
In this homework, we don't need the `derive_cpath1` and `derive_cpath2`.

Thanks.

**Comment by kaist-kihwan** [comment]
Then why are they in the skeleton code?
Can I just erase them?

**Comment by sujin0529** [comment]
Oh, it's unnecessary code, but we couldn't erase it.
Sorry for the confusion.
You can erase it! :)
